index.html

```diff
@@ -39,7 +40,166 @@ <h2 id="description-text">I'm a Software Developer</h2>
                          
                 </div>
             </div>  
-        </section>     
-    </main>
+        </section> 
+        <section id="work-section">
+                <section class="recent-work">
+                    <h2 id="recent-work-title">My Recent Works</h2>
+                    <hr>
+                    <img id="img-project" src="images/Img-Placeholder.png" alt="images-work">
+                    <h3 id="project-description-title">Multi-Post Stories</h3>
+                    <p id="project-description">A daily selection of privately personalized reads;
+                       no accounts or sign-ups required. has been the industry's standard dummy
+                       text ever since the 1500s, when an unknown printer took a standard dummy text.
+                    </p>
+                    <div class="button-section">
+                        <button class="btn-work">CSS</button>
+                        <button class="btn-work">HTML</button>
+                        <button class="btn-work">Bootstrap</button>
+                        <button class="btn-work">Ruby</button>
+                        <button class="view-button">See project</button>
+                    </div>       
+                    </section>
+                <section id="card-section">
+                    <div class="post">
+                        <div class="card-contents">
+                            <h3 class="card-main-title">Profesional Art Printing Data</h3>
+                            <p class="card-description">A daily selection of privately personalized reads;
+                               no accounts or sign-ups required. Has been the industry's standard.
+                            </p>
+                            <div class="card-buttons">
+                               <button class="btn-card btn-work">HTML</button>
+                               <button class="btn-card btn-work">Bootstrap</button>
+                               <button class="btn-card btn-work">Ruby</button>
+                            </div>
+                            
+                        </div> 
+                        <button class="card-big-btn" >See project</button>    
```

Comment:
Please kindly Implement the button interactions (enable, hover, etc.). According to the project requirements and stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 💪

Figma
![image](https://user-images.githubusercontent.com/78251927/227320571-eae8aab4-ddd0-4f5d-aeac-7793dab01540.png)